File: C Programs/area.c
#include <stdio.h> // header files
#include <conio.h>
int main()
{ 
    int base, height, radius;
    float area_tri, area_cir;
    printf("Enter the Radius of circle: ");
    scanf("%d", &radius);
    area_cir = 3.14*radius*radius;
    printf("Area of Circle is: %.2f \n", area_cir);

    printf("Enter the base of triangle: ");
    scanf("%d", &base);
    printf("Enter the height of triangle: ");
    scanf("%d", &height);
    area_tri = 0.5*base*height;
    printf("Area of Triangle is: %.2f \n", area_tri);


}

File: Java/area.java
package Java;
import java.util.Scanner;

public class area {
    public static void main(String[] args)
    {
        int radius, base, height;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the radius of circle: ");
        radius = sc.nextInt();
        double area_cir = 3.14 *radius*radius;
        System.out.println("Area of circle is: "+ (float)area_cir);
        
        System.out.print("Enter the base of triangle: ");
        base = sc.nextInt();
        System.out.print("Enter the height of triangle: ");
        height = sc.nextInt();
        double area_tri = 0.5*base*height;
        System.out.println("Area of Triangle is: "+ (float)area_tri);

    }
}
